pkg/router/template/template_helper_test.go

```diff
@@ -1046,9 +1053,46 @@ func TestParseIPList(t *testing.T) {
 			expectedEmpty: true,
 		},
 		{
-			name:          "Wrong IP in a list",
-			input:         "192.168.1.0 2001:0db8:85a3:0000:0000:8a2e:0370:7334 172.16.14.10/24 2001:0db8:85a3::8a2e:370:10/64 64:ff9b::192.168.0.1 10.",
-			expectedEmpty: true,
+			name:           "Wrong IPv4 in an IPs only list",
+			input:          "192.168.1.0 2001:0db8:85a3:0000:0000:8a2e:0370:7334 172.16.14.10/24 2001:0db8:85a3::8a2e:370:10/64 64:ff9b::192.168.0.1 10.",
+			expectedEmpty:  false,
```

Comment:
When `expectedEmpty` is `false` it can be skipped from the test struct, similar to the other tests which were modified above.